File: hrms/src/main/java/com/konstantinlevin77/hrms/business/abstracts/EmployerService.java
package com.konstantinlevin77.hrms.business.abstracts;

import java.util.List;

import com.konstantinlevin77.hrms.core.results.abstracts.DataResult;
import com.konstantinlevin77.hrms.core.results.abstracts.Result;
import com.konstantinlevin77.hrms.entities.concretes.Employer;
import com.konstantinlevin77.hrms.entities.dtos.EmployerWithoutJobPostingDto;

public interface EmployerService {
	
	public DataResult<List<Employer>> getAll();
	
	public Result add(EmployerWithoutJobPostingDto employer);
	
	public Result verifyEmail(int id);

}

File: hrms-frontend/src/services/EmployerService.js
import axios from 'axios'

export default class EmployerService{

    getAll() {
        return axios.get("localhost:8080/api/employer/getall");
    }

    add(employer) {
        return axios.post("localhost:8080/api/employer/add",employer)
    }

    verifyEmail(id) {
        return axios.post("localhost:8080/api/employer/verifyEmail",id)
    }

}